keras/engine/training.py

```diff
@@ -1932,7 +1940,8 @@ def evaluate_generator(self, generator, steps,
         try:
             if is_sequence:
                 enqueuer = OrderedEnqueuer(generator,
-                                           use_multiprocessing=use_multiprocessing)
+                                           use_multiprocessing=use_multiprocessing,
+                                           ordered=ordered)
```

Comment:
Does it make sense?  evalute_generator(...,ordered=True) ==evalute_generator(...,ordered=False)